### PR TITLE
Small changes in string manupulation

### PR DESCRIPTION
**Pull request by ghost**
References #6 and #7

### CODE REVIEW

File: Ragnarok/Bragi/Bragi.cpp [hunk 1]
```diff
@@ -8,32 +8,28 @@
...
 	}
 
 	std::vector<std::string> ListDir(char* dir) // dir not used yet.
 	{
 		std::vector<std::string> contents;
-		const char* path = Path();
+		std::string path = Path();
 		WIN32_FIND_DATA data;
-		HANDLE hFind = FindFirstFile("C:\\Users\\Hive\\Documents\\*", &data);      // DIRECTORY
+		HANDLE hFind = FindFirstFile((LPCSTR)Path().c_str(), &data);      // DIRECTORY
```
Comment (DigitMan27) [inline Ragnarok/Bragi/Bragi.cpp]: You use the `(LPSTR)Path().c_str()` in this line as a result the `std::string path = Path()` become a unused variable.

**Comment by ghost** [comment]
@DigitMan27 Let me know what you think of these changes.

File: Ragnarok/Bragi/Bragi.cpp (hunk 1 above)
Comment (ghost) [reply Ragnarok/Bragi/Bragi.cpp]: You are correct. I will fix this!

**Comment by ghost** [comment]
@DigitMan27 I removed some unused variables

**Review by DigitMan27 — APPROVED** [review]

**Pull request merged**